images/macos/helpers/Xcode.Installer.psm1

```diff
@@ -30,7 +31,13 @@ function Invoke-DownloadXcodeArchive {
 
     # TO-DO: Consider replacing of xcversion with own implementation
     Write-Host "Downloading Xcode $resolvedVersion"
-    Invoke-ValidateCommand "xcversion install '$resolvedVersion' --no-install"
+    Invoke-ValidateCommand "xcversion install '$resolvedVersion' --no-install" | out-host
+    $fixedResolvedVersion = "$resolvedVersion" -replace " ", "_"
+    $xcodeXip = Get-ChildItem -Path $DownloadDirectory -Filter "Xcode_$fixedResolvedVersion.xip" | Select-Object -First 1
+    $tempXipDir = New-Item -Path $DownloadDirectory -Name "Xcode$fixedResolvedVersion" -ItemType "directory"
+    Move-Item -Path "$xcodeXip" -Destination $tempXipDir
+
+    return "$tempXipDir"
```

Comment:
Why do we need quotes here and in 38 lines?

Reply:
Done.